account_bank_statement_import_camt_details/models/camt_parser.py

```diff
@@ -41,6 +41,13 @@ def parse_transaction_details(self, ns, node, transaction):
                 './ns:TwnNm', namespaces={'ns': ns})
             if city_node:
                 address_values['city'] = city_node[0].text
+            generic_nodes = address_node[0].xpath(
+                './ns:AdrLine', namespaces={'ns': ns})
+            address_values.update({
+                'address_line_' + str(i): node.text
```

Comment:
You may want to update [these examples](https://github.com/OCA/bank-statement-import/pull/199/files#diff-eb6787e4d3e1251b117823745cd8d271R68) with your new keys 'address_line_xxxx'.

About the order of things: I don't know if `AdrLine` and eg. `TwnNm` sometimes appear in the same statement, but in case it happens, I would add these `Adrline` fields between `Street` and `Zip` and not at the end. Otherwise it would look weird.